vite-template/.gitignore

```diff
@@ -0,0 +1,27 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/cypress/screenshots
+/cypress/videos
+/coverage
+.nyc_output
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+
+# editors
+.vscode
```

Comment:
Maybe we can add `.idea` here as well? It is popular IDE

Reply:
0aac8fc ✅ 

Initially, I added `.vscode` because I configured a different color set for this project (as testing manually requires opening other instances of VS Code, I sometimes got confused between which tab is the template and which tab is the generated app ><).